src/Providers/ReviewsCoUk/Service.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\ReviewsCoUk;
+
+use GuzzleHttp\Client;
+use PodPoint\Reviews\Service as ServiceInterface;
+
+class Service implements ServiceInterface
+{
+    /**
+     * A HTTP client instance.
+     *
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * A reviews.co.uk configuration instance.
+     *
+     * @var Configuration
+     */
+    private $config;
+
+    /**
+     * Sets the client and configuration for the service.
+     *
+     * @param Client $client
+     * @param Configuration $config
+     */
+    public function __construct(Client $client, Configuration $config)
+    {
+        $this->client = $client;
+        $this->config = $config;
+    }
+
+    /**
+     * Creates an invite.
+     *
+     * @param array $options
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function invite(array $options)
+    {
+        $options = new InviteOptions($options);
+
+        $this->request('POST', 'merchant/invitation', [
+            'form_params' => [
+                'name' => $options->name,
+                'email' => $options->email,
+                'order_number' => $options->orderNumber,
+            ],
+        ]);
+    }
+
+    /**
+     * Get review(s).
+     *
+     * @param array $options
+     *
+     * @return array|null
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function get(array $options)
+    {
+        $options = new GetOptions($options);
+
+        $parameters = [
+            'store' => $this->config->store,
+        ];
+
+        if ($options->hasOrderNumber()) {
+            $parameters['apikey'] = $this->config->apiKey;
```

Comment:
is this suppose to go as a query param? 🤔

Reply:
Hmm, let me double check

Reply:
To be honest, I'm not sure we need that there. As we send it inside the header of the request.